docs/application/native/guides/ui/tizen-ws-shell/index.md

```diff
@@ -0,0 +1,73 @@
+# Tizen Window System Shell
+
+The Tizen Window System Shell (TZSH) is an interface for manipulating windows of the system Graphical User Interface (GUI) services such as the Quickpanel and the Softkey. It provides C libraries that allow you to control windows of system GUI services and get notifications about the state changes of each window within an application.
```

Comment:
As per @sabeena21's comment, please remove "the" before "Quickpanel" and "Softkey" and also add "the" before "system GUI services".


For more information, see https://github.com/Samsung/tizen-docs/pull/771#discussion_r302809074

Reply:
Done.